ml-completion/src/main/kotlin/RustMLRankingProvider.kt

```diff
@@ -0,0 +1,15 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ml
+
+import com.intellij.internal.ml.catboost.CatBoostJarCompletionModelProvider
+import com.intellij.lang.Language
+
+@Suppress("UnstableApiUsage")
+class RustMLRankingProvider : CatBoostJarCompletionModelProvider("Rust", "rust_features", "rust_model") {
```

Comment:
It worth adding a test to check that metadata we ship with the model is fully consistent. See example here -  `com.jetbrains.completion.ml.ranker.MetadataConsistencyTest#testRustMetadata`